operators/advanced_network/adv_network_types.h

```diff
@@ -188,7 +188,7 @@ inline AnoMgrType manager_type_from_string(const std::string& str) {
   if (str == ANO_MGR_STR__DOCA) return AnoMgrType::DOCA;
```

Comment:
Change the enum too?

Reply:
either, I feel like that one can wait until we rename the actual manager `DocaMgr`, and mapping from gpunetio as a string to doca as a manager is consistent. but fine either way, that one is low impact also